hooks/sdk-fork-fix.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * This hook is designed to address issues with forking of the projects correct SDK when the build
+ * or clean commands are called with both a --project-directory option
+ *
+ * @copyright
+ * Copyright (c) 2009-2020 by Appcelerator, Inc. All Rights Reserved.
+ *
+ * @license
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+exports.cliVersion = '>=3.2';
+
+exports.init = function (logger, config, cli, _appc) {
+	function getSDK() {
+		return (cli.sdk && (cli.sdk.manifest && cli.sdk.manifest.version || cli.sdk.name)) || (cli.manifest && cli.manifest.version);
+	}
+
+	function handleFork (data, commandName) {
+		var sdk = getSDK();
```

Comment:
```suggestion
		const sdk = getSDK();
```

Reply:
But also, is this even used? I assume we're actually just checking that `cli.sdk` is "valid"? Because below we just use `cli.sdk.path`. If so, maybe rename the function to better reveal it's use.

Reply:
Looks like it was used to get a semver compatible version for the SDK so that we could safely compare versions (part of [TIMOB-16052](https://jira.appcelerator.org/browse/TIMOB-16052)). We don't need to compare anymore so I'll nuke it